Microsoft.Azure.CosmosRepository.Samples/WebTier/Controllers/LanguageController.cs

```diff
@@ -35,8 +35,21 @@ public class LanguageController : ControllerBase
         public ValueTask<Language> PutLanguage([FromBody] Language language) =>
             _repository.UpdateAsync(language);
 
+        [HttpPut("{id}")]
+        public async ValueTask<IActionResult> UpdateName([FromRoute] string id, [FromQuery] string name)
+        {
+            if (string.IsNullOrWhiteSpace(name))
+            {
+                return BadRequest("A name is required");
+            }
+
+            await _repository.UpdateAsync(id, id, builder => builder.Replace(language => language.Name, name));
```

Comment:
```suggestion
            await _repository.UpdateAsync(
                id, builder: b => b.Replace(language => language.Name, name));
```

Once the `UpdateAsync` API is updated to accept a `null` for the `partionKeyValue` then you could do this, thinking about it a bit more - we might want to change the order to `id, builder, partionKeyValue, cancallationToken`

Reply:
Yes, we will have to re-jig the parameters.

Reply:
Have amended this.